src/System.Net.Security/ref/System.Net.Security.cs

```diff
@@ -160,6 +161,7 @@ public partial class SslStream : AuthenticatedStream
         public override int EndRead(IAsyncResult asyncResult) { throw null; }
         public override void EndWrite(IAsyncResult asyncResult) { }
         public override void Flush() { }
+        public override System.Threading.Tasks.Task FlushAsync(System.Threading.CancellationToken cancellationToken) { throw null; }
```

Comment:
The src changes look good.

I think technically these ref changes are considered new APIs, even though they're just overrides, and need to be ifdef'd for netcoreapp11. @weshaggard?